stream-helpers/map.ts

```diff
@@ -0,0 +1,33 @@
+import type { Operation, Stream } from "effection";
+
+/**
+ * Transforms each item in the stream using the provided function.
+ *
+ * @param fn - The function to transform each item
+ * @returns A stream transformer that applies the function to each item
+ */
+export function map<A, B>(
+  fn: (value: A) => Operation<B>,
+): (stream: Stream<A, never>) => Stream<B, never> {
+  return function (stream: Stream<A, never>): Stream<B, never> {
```

Comment:
mapping should work on all streams, not just `TClose = never`

Reply:
Ok